PolyEngine/RenderingDevice/OpenGL/Src/Proxy/imgui_impl_opengl3.h

```diff
@@ -0,0 +1,37 @@
+// ImGui Renderer for: OpenGL3 / OpenGL ES2 / OpenGL ES3 (modern OpenGL with shaders / programmatic pipeline)
```

Comment:
I would keep it as example of custom rendering pass introduced by engine system

Reply:
But this is not custom pass. This is third party code injected into our code tree. I think it would be much better to separate this. I know it's easier that way, but I would still try. You can think of it as something similar to epoxy, it provides gl bindings to our engine.

Reply:
Need to compare OpenGL implementation with Vulkan.